site/reference/pkg/diff/README.md

```diff
@@ -19,59 +19,6 @@ It can display differences between:
 The diff tool can be specified.  By default, the local 'diff' command is used to
```

Comment:
I have updated the section and added a TEMPLATE.md file in the reference doc root directory.

For the diff command, it is actually a bit challenging since the command relies on a command line diffing tool (and the default tool for osx/linux is called `diff`). I have tried to rephrase the document to make it clearer which tool we are referring to.
On somewhat separate issue is that the implementation of diff probably have challenges on windows and ideally we want to avoid relying on specific binaries being available on the path. But this doc tries to explain how the diff command works today.